demo/angular/README.md

```diff
@@ -0,0 +1,51 @@
+## Angular wrapper example
+
+### Usage
+
+```typescript
+gridstackConfig: GridStackOptions = {
```

Comment:
this is very specific to your needs... might not be best generic example, but easy enough for me to cleanup.

Reply:
> this is very specific to your needs... might not be best generic example, but easy enough for me to clean up.

Like it's in the README.md caveat section